docs/resources/elasticsearch_script.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Cluster"
+layout: ""
+page_title: "Elasticstack: elasticstack_elasticsearch_script Resource"
+description: |-
+  Creates or updates a stored script or search template.
+---
+
+# Resource: elasticstack_elasticsearch_script
+
+Creates or updates a stored script or search template. See https://www.elastic.co/guide/en/elasticsearch/reference/current/create-stored-script-api.html
+
+## Example Usage
+
+```terraform
+provider "elasticstack" {
+  elasticsearch {}
+}
+
+resource "elasticstack_elasticsearch_script" "my_script" {
+  script_id = "my_script"
+  source    = "Math.log(_score * 2) + params['my_modifier']"
+  context   = "score"
```

Comment:
This example is no longer valid, and needs `lang` defined.

Reply:
Oh good catch, fixed🙏